.travis.yml

```diff
@@ -21,11 +21,12 @@ addons:
     - protobuf-compiler
     - python-protobuf
 before_install:
-  - export CHROME_BIN=google-chrome
   - export DISPLAY=:99.0
   - sh -e /etc/init.d/xvfb start
   - wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
-  - sudo dpkg -i google-chrome*.deb
+  - mkdir chrome
```

Comment:
I think we should be explicit here and create the directory under $TRAVIS_BUILD_DIR

`mkdir $TRAVIS_BUILD_DIR/chrome`